packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -12,6 +12,13 @@ import { TelemetryDataTag } from "@fluidframework/telemetry-utils";
 import { ContainerRuntime } from "./containerRuntime";
 import { DataStores } from "./dataStores";
 
+/**
+ * @deprecated - This will be removed once https://github.com/microsoft/FluidFramework/issues/9127 is fixed.
```

Comment:
Does this need a 'doc' update?

Reply:
I was thinking about it and I thought we don't need to advertise it since bindToContext was already deprecated.

Reply:
I agree but I thought the docs task would complain. Turns out it doesn't. :D